library/Garp/Functional/Modulo.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @package  Garp\Functional
+ * @author   Harmen Janssen <harmen@grrr.nl>
+ * @license  https://github.com/grrr-amsterdam/garp-functional/blob/master/LICENSE.md BSD-3-Clause
+ */
+namespace Garp\Functional;
+
+/**
+ * Modulo of two numbers.
+ *
+ * @param int $left
+ * @param int $right
+ * @return int
+ */
+function modulo($left, $right = null) {
+    $moduler = function ($right) use ($left) {
+        return $left % $right;
```

Comment:
See also my comment on `divide`. It makes more sense to me to do `$right % $left`.